tests/_modules.js

```diff
@@ -0,0 +1,715 @@
+module.exports =  [{ path: '/Users/samrichard/Development/plugabilly/node_modules/ava',
```

Comment:
@Snugug 

Is this set up because you can't use the repo's own package.json because all the deps are in devDeps?